solidity/interfaces/IStorageMirror.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.19;
+
+interface IStorageMirror {
+  /**
+   * @notice Emits when the settings have been updated
```

Comment:
We should have separator sections for events, structs etc  check here: https://www.notion.so/defi-wonderland/Coding-Style-512e76bd36374a90882496b230b3ad92